src/button/index.css

```diff
@@ -98,17 +103,15 @@ governing permissions and limitations under the License.
   border-style: solid;
   border-radius: var(--spectrum-button-primary-border-radius);
 
-  height: var(--spectrum-button-primary-height);
+  min-height: var(--spectrum-button-primary-height);
+  height: auto;
   min-width: var(--spectrum-button-primary-min-width);
 
-  padding: 0 calc(var(--spectrum-button-primary-padding-x) - var(--spectrum-button-primary-border-size)) 0;
+  padding: var(--spectrum-button-padding-y) calc(var(--spectrum-button-primary-padding-x) - var(--spectrum-button-primary-border-size));
 
   font-size: var(--spectrum-button-primary-text-size);
   font-weight: var(--spectrum-button-primary-text-font-weight);
 
-  /* Prevent vertical text alignment mismatch between anchor and button in Chrome */
-  line-height: 0;
```

Comment:
This is the hack that was making text vertically align in Chrome... We'll have to either find another hack or live with it. Changing font-size to 14px also makes it align nicely.
![image](https://user-images.githubusercontent.com/201344/58207570-cb0c5600-7c97-11e9-9d2f-45dedfd9b18c.png)

Reply:
Filed https://github.com/adobe/spectrum-css/issues/185 to track this